app/src/main/java/com/mychefassistant/presentation/kitchen/insert/KitchenInsertFragment.kt

```diff
@@ -29,19 +48,28 @@ class KitchenInsertFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        titleInput = view.findViewById(R.id.title_input)
-        titleInputLayout = view.findViewById(R.id.title_layout)
-        locationInput = view.findViewById(R.id.location_input)
+        titleInput = view.findViewById(R.id.fragment_kitchen_insert_title_layout_input)

```

Comment:
Are you using data binding or not? If yes why findViewById? If no dont use it anywhere.
You get to use one thing in the project.